test/miscellaneous.tests.js

```diff
@@ -298,7 +299,7 @@ contract("LiquidityBridgeContract", async (accounts) => {
     let destAddr = accounts[1];
     let fedBtcAddress = "0x0000000000000000000000000000000000000000";
     let liquidityProviderRskAddress = accounts[0];
-    let data = web3.eth.abi.encodeFunctionCall(mock.abi[0], [
+    let data = web3.eth.abi.encodeFunctionCall(mock.abi[1], [
       "100100001110010",
```

Comment:
what is the string?

Reply:
same situation of first comment, I don't know the meaning of the value, just updated the index